client/login/magic-login/index.jsx

```diff
@@ -956,9 +957,14 @@ class MagicLogin extends Component {
 				<div className="grav-powered-magic-login__content">
 					<img src={ oauth2Client.icon } width={ 27 } height={ 27 } alt={ oauth2Client.title } />
 					<RequestLoginEmailForm
-						flow={ oauth2Client.name }
+						flow={ oauth2Client.name ?? oauth2Client.source }
```

Comment:
Seeing this in multiple places makes me wonder if we should have a helper for it 🤔 

I also wonder if we should check if this is a hardcoded client (instead of checking if the name property exists). Checking the name could prove flimsy in the future. Wdyt?

Reply:
✅ Sounds good, it has been addressed through [280b39e](https://github.com/Automattic/wp-calypso/pull/93133/commits/280b39e013ac391a49e6b6305c8754377e7926dd)